_includes/figure.html

```diff
@@ -9,20 +12,24 @@
     {%- endif %}>
     <picture>
       {%- if include.name-mobile %}
-      <source media="(max-width: 767px)" srcset="/assets-local/figures/{{ page.slug }}/{{ include.name-mobile }}" />
+      <source media="(max-width: 575px)" srcset="/assets-local/figures/{{ page.slug }}/{{ include.name-mobile }}" />
       {%- endif %}
       <img src="/assets-local/figures/{{ page.slug }}/{{ include.name }}" alt="{{ include.alt }}">
     </picture>
   </a>
   {%- if include.caption or include.source-text %}
-  <div class="source fancybox-custom-caption">
+  <div class="fancybox-custom-caption">
     {%- if include.caption %}
-      {{- include.caption }} 
+      <div class="caption caption-below">
+        {{- include.caption | markdownify }}
+      </div>
     {%- endif %}
     {%- if include.source-text %}
-      Zdroj: {%- if include.source-url %}<a href="{{ include.source-url }}">{% endif %}
-        {{ include.source-text }}
-        {%- if include.source-url %}</a>{% endif %}
+      <div class="source">
+        Zdroj: {%- if include.source-url %}<a href="{{ include.source-url }}">{% endif %}
```

Comment:
We should probably localise the "Zdroj" later.

Reply:
Let me do it right away (as we anyway introduce new locale strings in this PR)